yaml/annotations.adoc

```diff
@@ -136,3 +136,39 @@ Applies to:
 // Applies to:
 // * StatefulSets
 
+
+// Type: procedure
+[id="service-yaml"] 
+= Creating a service when service-sync is disabled
+
+If you set `service-sync: "false"` when creating a site and you want to consume a service exposed on another site:
+
+. Create `backend.yaml` similar to the following:
++
+[source, yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  name: backend
+  annotations:
+    skupper.io/proxy: tcp
+    skupper.io/address: backend
+    skupper.io/port: "8080"
+    skupper.io/ingress-only: "true"
+spec:
+  ports:
+    - name: http
+      protocol: TCP
+      port: 8080
+      targetPort: 8080
```

Comment:
There are three places where 'port' is set; it might be interesting to explain what's the role of each of them, especially given that one of them will be overwritten by Skupper.